tests/integration/netapi/rest_tornado/test_app.py

```diff
@@ -36,6 +37,11 @@ def mod_opts(self):
 @skipIf(HAS_ZMQ_IOLOOP is False, 'PyZMQ version must be >= 14.0.1 to run these tests.')
 @skipIf(StrictVersion(zmq.__version__) < StrictVersion('14.0.1'), 'PyZMQ must be >= 14.0.1 to run these tests.')
 class TestSaltAPIHandler(_SaltnadoIntegrationTestCase):
+
+    def setUp(self):
+        super(TestSaltAPIHandler, self).setUp()
+        os.environ['ASYNC_TEST_TIMEOUT'] = str(300)
```

Comment:
Why not `os.environ['ASYNC_TEST_TIMEOUT'] = "300"` ?

Reply:
Valid point